packages/tools/client-debugger/client-debugger/src/DevtoolsLogger.ts

```diff
@@ -91,40 +96,31 @@ export class DevtoolsLogger extends TelemetryLogger {
 	// #endregion
 
 	/**
-	 * Create an instance of this logger
-	 * @param namespace - Telemetry event name prefix to add to all events
-	 * @param properties - Base properties to add to all events
+	 * Creates a new DevtoolsLogger, which will post telemetry events to the Window, and will forward them to the
+	 * provided baseLogger (if one is provided).
+	 *
+	 * @param baseLogger - (optional) Base logger to which all telemetry events will be forwarded (in addition to
+	 * posting them to the Window).
+	 * @param namespace - Telemetry event name prefix to add to all events.
+	 * @param properties - Base properties to add to all events.
 	 */
 	public static create(
+		baseLogger?: ITelemetryBaseLogger,
 		namespace?: string,
 		properties?: ITelemetryLoggerPropertyBags,
 	): DevtoolsLogger {
-		return new DevtoolsLogger(namespace, properties);
-	}
-
-	/**
-	 * Mix in this logger with another.
-	 * The returned logger will output events to the newly created DevTools extension logger *and* the base logger.
-	 * @param namespace - Telemetry event name prefix to add to all events
-	 * @param baseLogger - Base logger to output events (in addition to DevTools extension logger being created). Can be undefined.
-	 * @param properties - Base properties to add to all events
-	 */
-	public static mixinLogger(
-		namespace?: string,
-		baseLogger?: ITelemetryBaseLogger,
-		properties?: ITelemetryLoggerPropertyBags,
-	): TelemetryLogger {
 		if (!baseLogger) {
-			return DevtoolsLogger.create(namespace, properties);
+			return new DevtoolsLogger(namespace, properties);
 		}
 
-		const multiSinkLogger = new MultiSinkLogger(undefined, properties);
-		multiSinkLogger.addLogger(
-			DevtoolsLogger.create(namespace, this.tryGetBaseLoggerProps(baseLogger)),
-		);
-		multiSinkLogger.addLogger(ChildLogger.create(baseLogger, namespace));
+		// TODO: what is this for?
+		const devtoolsLoggerProperties = properties ?? this.tryGetBaseLoggerProps(baseLogger);
```

Comment:
It's originally a copy of the logic in DebugLogger. Since this one doesn't really need to enrich the events, I think we could remove the properties and the namespace from this one's creation.

Reply:
Sounds good! Will update.